integration_tests/poetry.lock

```diff
@@ -123,7 +123,7 @@ mnemonic = ">=0.19, <0.20"
 
 [package.source]
 type = "git"
-url = "https://github.com/crypto-com/cryptopy.git"
+url = "https://github.com/crypto-org-chain/chainlibpy.git"
 reference = "fe444873294ed47de505596e307c4311c1489beb"
```

Comment:
was this updated by hand, as the Git repo is no longer referenced in pyproject.toml? Not a python expert, but I guess this lock file can be updated with `poetry update` -- https://python-poetry.org/docs/cli/#update ? @yihuang @linfeng-crypto

Reply:
ok, blocked by https://github.com/crypto-org-chain/chain-main/issues/455

Reply:
>> ok, blocked by #455

yap

Reply:
it's resolved now